src/net/sourceforge/kolmafia/request/UseSkillRequest.java

```diff
@@ -583,19 +585,6 @@ public long getMaximumCast() {
           int birds = Preferences.getInteger("_birdsSoughtToday");
           return (birds < 7 && option != 1) ? 6 - birds : Long.MAX_VALUE;
         }
-
-      case SkillPool.INVISIBLE_AVATAR:
```

Comment:
this block has gone away and doesn't seem to be replaced with anything: does what it did still work?

Reply:
Was there a before and after test demonstrating this?

Can there be?